src/pages/instances/actions/InstanceBulkAction.tsx

```diff
@@ -116,13 +116,22 @@ const InstanceBulkAction: FC<Props> = ({
       confirmMessage={
         <>
           {selectedSummary}
-          {getLineOrder().map((state) => statusLine(state, action))}
-          <br />
-          Are you sure you want to {confirmLabel.toLowerCase()}{" "}
-          {instanceActions(instances, action).length === 1
-            ? "this instance"
-            : "these instances"}
-          ?
+          {actionLines}
+          {actionLines.filter((elem) => elem !== null).length === 1 && (
+            <div className="affected-items">
+              <ExpandableList
```

Comment:
The small font list looks a bit odd in the modal. I wonder if we should avoid the list here completely? Or don't show it initiially, but have a "show instance names" link that reveals them? Better get advice from a designer.